docs/sources/zendesk.md

```diff
@@ -0,0 +1,74 @@
+# Event source for Zendesk
+
+This event source registers itself as a notification receiver in Zendesk in order to capture events such as ticket
+creations.
+
+## Prerequisite: API token
+
+An API token is required in order to let the TriggerMesh Zendesk event source create a corresponding [Target][zd-target]
+and [Trigger][zd-trigger] in your Zendesk account. To create a new API token from the Zendesk Admin interface, follow
+the instructions at [Generating a new API token ][zd-token].
+
+## Deploying an instance of the Source
+
+Open the Bridge creation screen and add a source of type `Zendesk`.
+
+![Adding a Zendesk source](../images/zendesk-source/create-bridge-1.png)
+
+In the Source creation form, give a name to the event source and add the following information:
+
+* **Email**: Email address associated with the Zendesk account.
+* [**Subdomain**][zd-subdom]: Name of the Zendesk subdomain, without the `zendesk.com` domain or `https://` scheme.
+* [**Token**][zd-token]: Reference to a [TriggerMesh secret][tm-secret] containing an API token to communicate with the
+  Zendesk API, as described in the previous section.
+* **Webhook username/password**: arbitrary user name and password, used to verify event callbacks.
+
+![Zendesk source form](../images/zendesk-source/create-bridge-2.png)
+
+After clicking the `Save` button, you will be taken back to the Bridge editor. Proceed to adding the remaining
+components to the Bridge, then submit it.
+
+![Bridge overview](../images/zendesk-source/create-bridge-3.png)
+
+A ready status on the main _Bridges_ page indicates that the Zendesk [Target][zd-target] and [Trigger][zd-trigger] were
+successfully created and that the event source is ready to forward events from Zendesk.
+
+![Bridge status](../images/bridge-status-green.png)
+
+## Verification of external resources
+
+To verify the successful deployment of the Zendesk event source, navigate to the [Targets][zd-target] tab of the
+_Extensions_ screen in the Zendesk Admin interface, below the _Settings_ section. The event source instance should have
+created a Target following the naming pattern `io.triggermesh.zendesksource.<user namespace>.<source name>`.
+
+![Zendesk Target name](../images/zendesk-source/targets-1.png)
+
+The Target is configured to include the configured webhook username and password in each request header.
```

Comment:
"The Target is configured to include the configured webhook username and password in each request header."

?*"The Target is configured to include the webhook username and password in each request header."

Reply:
Oops, didn't notice that one.

Reply:
Done in [157f525a](https://github.com/triggermesh/docs/compare/e2562255bfd388b37eb4535f3fb3e04efa21683e..157f525a4ea20d69c996b927b0ade05b1fa723ae)